var/spack/repos/builtin/packages/py-fparser/package.py

```diff
@@ -11,19 +11,25 @@ class PyFparser(PythonPackage):
 
     homepage = "https://github.com/stfc/fparser"
     git = "https://github.com/stfc/fparser.git"
-    pypi = "fparser/fparser-0.0.16.tar.gz"
+    pypi = "fparser/fparser-0.1.3.tar.gz"
 
     version("develop", branch="master")
+    version("0.1.3", sha256="10ba8b2803632846f6f011278e3810188a078d89afcb4a38bed0cbf10f775736")
     version("0.0.16", sha256="a06389b95a1b9ed12f8141b69c67343da5ba0a29277b2997b02573a93af14e13")
     version("0.0.6", sha256="bf8a419cb528df1bfc24ddd26d63f2ebea6f1e103f1a259d8d3a6c9b1cd53012")
     version("0.0.5", sha256="f3b5b0ac56fd22abed558c0fb0ba4f28edb8de7ef24cfda8ca8996562215822f")
 
+    depends_on("py-setuptools@42:", type="build", when="@0.1:")
     depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools-scm@6.2:+toml", type="build", when="@0.1:")
+    depends_on("py-wheel@0.29:", type="build", when="@0.1:")
+
+    depends_on("py-importlib-metadata", type=("build", "run"), when="@0.1:")
 
     depends_on("py-numpy", type=("build", "run"), when="@:0.0.5")
     depends_on("py-nose", type=("build", "run"), when="@:0.0.7")
     # six is unused as of 0.0.15, but still listed in setup.py
-    depends_on("py-six", type=("build", "run"), when="@0.0.6:")
+    depends_on("py-six", type=("build", "run"), when="@0.0.6:0.1")
```

Comment:
```suggestion
    depends_on("py-six", type=("build", "run"), when="@0.0.6:0.0")
```
Version ranges are inclusive

Reply:
I'm aware. So 0.1 also covers 0.1.3? What about using 0.1.0 instead here, just to clarify?

Reply:
Just in case, I used 0.0.16, since that is the last release that Spack supports.

Reply:
0.1 mean all 0.1.X versions. So both `:0.0` and `:0.0.16` are equivalent.